enterprise/models.py

```diff
@@ -65,6 +65,7 @@ class Meta(object):
 
     uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Enterprise Customer name."))
+    catalog = models.PositiveIntegerField(null=True, help_text=_("Course catalog for the Enterprise Customer."))
```

Comment:
@haikuginger nit/optional suggestion: it would be a bit more human-friendly if `catalog` field stored some kind of catalogue name, not ID. Not sure if it is possible with catalogue service though (i.e. if catalogue service has string keys).